std/range/primitives.d

```diff
@@ -2019,6 +2020,11 @@ equivalent to $(D empty(array)).
     auto a = [ 1, 2, 3 ];
     assert(!a.empty);
     assert(a[3 .. $].empty);
+
+    int[string] b;
```

Comment:
For future PRs: documented examples should be as minimal and viable as possible, i.e. they should be split up whenever possible (like here).
Sorry that I didn't notice this before.